scenarios/kubernetes_e2e.py

```diff
@@ -334,6 +356,13 @@ def main(args):
     # So if you overwrite FOO=c for a local run it will take precedence.
     #
 
+    if (args.image_family or args.image_project) and args.mode == 'docker':
+        raise ValueError(
+            '--image-family / --image-project is not supported in docker mode')
+    if bool(args.image_family) != bool(args.image_project):
+        raise ValueError(
+            '--image-family and --image-project must be both set or unset')
```

Comment:
this logic should probably in `bootstrap_test.py`, maybe [here](https://github.com/kubernetes/test-infra/blob/master/jenkins/bootstrap_test.py#L2206), we don't want people check in such combination and get a busted scenario run